pkgs/by-name/vo/voidmap/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  pname = "voidmap";
+  version = "1.1.5-unstable-2023-09-13";
+
+  src = fetchFromGitHub {
+    owner = "void-rs";
+    repo = "void";
+    rev = "ab32290632fa9477a7e59b884bdfa69fb4b91906";
+    hash = "sha256-+P83psu+BYzgC/I/Ul7vrbZ99KIybd410/ycsIY1pGI=";
+  };
+
+  cargoHash = "sha256-3pObVbtvT6rL7JZ3DyTWenVog5WdZno1BcgKt+WelFs=";
+
+  doCheck = false;
+
+  meta = {
+    description = "A terminal-based personal organizer";
+    homepage = "https://github.com/void-rs/void";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ poptart ];
+    platforms = lib.platforms.linux;
```

Comment:
Please add `mainProgram`.

Reply:
Whoops missed that in the required meta attributes. Added! :+1: